mne/epochs.py

```diff
@@ -1560,6 +1569,15 @@ def _project_epoch(self, epoch):
             epoch = np.dot(self._projector, epoch)
         return epoch
 
+    def _handle_empty(self, on_empty, msg):
+        if len(self.events) == 0:
+            msg = (
+                f"epochs.{msg}() can't run because this Epochs-object is empty.\n"
```

Comment:
Remove newline (and use new var) as it seems unnecessary here
```suggestion
                f"epochs.{meth}() can't run because this Epochs-object is empty. "
```